test/qa/test_settings.py

```diff
@@ -3,32 +3,26 @@
 import timeout_decorator
 
 import lib
-from lib import (
-    daemon,
-    info,
-    logging,
-    login,
-    network,
-)
+from lib import daemon, dns, info, logging, login, meshnet, network, settings
 
 
 def setup_module(module):  # noqa: ARG001
     daemon.start()
-    login.login_as("default")
 
 
 def teardown_module(module):  # noqa: ARG001
-    sh.nordvpn.logout("--persist-token")
     daemon.stop()
 
 
 def setup_function(function):  # noqa: ARG001
     logging.log()
+    login.login_as("default")
```

Comment:
wouldn't be good to set to defaults before logging in, in case the tests are already logged in and mesh is on?